openedx/core/djangoapps/util/helpers.py

```diff
@@ -0,0 +1,48 @@
+"""
+Common helpers methods for django apps.
+"""
+
+import logging
+
+from django.core.exceptions import ImproperlyConfigured
+from provider.oauth2.models import AccessToken, Client
+from provider.utils import now
+
+
+log = logging.getLogger(__name__)
+
+
+def get_id_token(user, client_name):
```

Comment:
Was there a test left behind in edxnotes for this?  If so it needs to come along.  If not, please create a JIRA ticket to add tests for this function.